mimesis/builtins/nl.py

```diff
@@ -0,0 +1,35 @@
+from mimesis.builtins.base import BaseSpecProvider
+
+
+class NLSpecProvider(BaseSpecProvider):
+    """Provides data specific to nl locale"""
+
+    class Meta:
+        name = 'nl_provider'
+
+    def bsn(self) -> str:
+        """Generate a random, but valid burgerservicenummer (BSN).
+
+        :returns: random BSN
+
+        :Example:
+            255159705
+        """
+
+        # a bsn is valid when its 9 digits pass the '11 test'
+        def is_valid_bsn(number):
```

Comment:
This method must be private: `__is_valid_bsn`

Reply:
You're right, although I believe this should be a single leading underscore `_is_valid_bsn` for non-public functions, right?

Reply:
@jorisdevrede Absolutely.

Reply:
Done :-)